pkg/auth/csp/tanzu.go

```diff
@@ -476,13 +518,37 @@ func GetTanzuHubEndpointForTAP(orgID, accessToken string, useStagingIssuer bool)
 		return "", errors.Wrap(err, "could not unmarshal the services response")
 	}
 
+	var productIdentifier, displayName string
+	_ = centralConfigReader.GetCentralConfigEntry(centralConfigTanzuPlatformCSPProductIdentifierKey, &productIdentifier)
```

Comment:
So unlike prior to the change, central config is a requirement for getting hub endpoint now? 
Would it have been better if we can retain some default values for these, as the ultimate fallback?

Reply:
Yeah makes sense. I will update the PR accordingly.

Reply:
Updated